students/Roy_Tate/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,98 @@
+# author: githubtater
+
+def fizz_buzz():
+    '''A classic, simple computer science problem'''
+    current_number = 1
+    while current_number < 101:
```

Comment:
instead of having to keep track of counter vars, you can use `range` function that can generate the sequence for you.